man/ocitools-generate.1.md

```diff
@@ -137,18 +137,19 @@ read the configuration from `config.json`.
   The special *PATH* `host` removes any existing PID namespace from
   the configuration.
 
-**--poststart**=CMD
-  Path to command to run in poststart hooks. This command will be run before
-  the container process gets launched but after the container environment and
-  main process has been created.
+**--poststart**=[CMD[:ARG]]
```

Comment:
> **--bind**=*[[HOST-DIR:CONTAINER-DIR][:OPTIONS]]*
bind also used brackets around all. Based on your opinion, it's also need to fix?